src/gemm.jl

```diff
@@ -132,6 +138,7 @@ end
 end
 
 @inline VectorizationBase.vsum(x::Tropical{<:VecUnroll}) = Tropical(VectorizationBase.vmaximum(content(x)))
+@inline VectorizationBase.vsum(x::Tropical{<:Vec}) = Tropical(VectorizationBase.vmaximum(content(x)))
```

Comment:
Could do `Tropical{<:AbstractSIMD}` instead of having two separate but identical definitions.
That'd also support `Tropical{<:MM}`, for example.